src/common/commonutils/FileUtils.c

```diff
@@ -1608,15 +1623,16 @@ static int FindTextInCommandOutput(const char* command, const char* text, void*
     char* results = NULL;
     int status = 0;
 
-    if ((NULL == command) || (NULL == text))
+    if ((NULL == command) || (NULL == text) || (0 == strlen(command)) || (0 == strlen(text)))
     {
         OsConfigLogError(log, "FindTextInCommandOutput called with invalid argument");
         return EINVAL;
     }
 
-    if (0 == (status = ExecuteCommand(NULL, command, true, false, 0, 0, &results, NULL, log)))
+    // Execute this command with a 60 seconds timeout

```

Comment:
Please verify the SystemCommand function. I'm OK with explicitly specifying 60s here. But my problem is that in the mentioned function we have:
`int timeout = (timeoutSeconds > 0) ? timeoutSeconds : defaultCommandTimeout;`
which suggests when 0 is passed, the default timeout is used, but later on the function only validates the `timeoutSeconds` variable, which means no timeout is used which is misleading to me. If that's intended, please consider adding some comments there.

Reply:
That's a different purpose variable. The actual timeout is in timeoutSeconds. There is no default 60 secs timeout.